README.md

```diff
@@ -0,0 +1,40 @@
+# spine-doc-generator
+
+The command-line tool for Web Components documentation generating.
+[Polymer-analyzer](https://github.com/Polymer/polymer-analyzer) is used under the hood.
+
+## Installation
+```bash
+npm install -g git://github.com/SpineElements/spine-doc-generator.git
+```
+
+## Example
+
+### `spine-doc-generator [items-root]`
+If `items-root` omitted, the current directory is used as the root path.
+
+Generated docs location: `items-root/element-root/index.html`.
```

Comment:
What happens if there are older generated documentation files in place when we run this (it can be the typical scenario for us)?